public/content/whitepaper/index.md

```diff
@@ -12,7 +12,7 @@ _This introductory paper was originally published in 2014 by Vitalik Buterin, th
 
 _While several years old, we maintain this paper because it continues to serve as a useful reference and an accurate representation of Ethereum and its vision. To learn about the latest developments of Ethereum, and how changes to the protocol are made, we recommend [this guide](/learn/)._
 
-[Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.](./whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf)
+[Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.](/content/whitepaper/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf)
```

Comment:
For this to be properly fixed:
```suggestion
[Researchers and academics seeking a historical or canonical version of the whitepaper [from December 2014] should use this PDF.](/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf)
```

`/content/whitepaper` is already being added to the beginning through the `Link` component and an [isInternalPdf check](https://github.com/ethereum/ethereum-org-website/blob/0f4ae79ba3a8a4b00cf3d3bed422a43c954932ef/src/components/Link.tsx#L74)

Reply:
Hey @lucas-amberg! Thanks for the PR... I'll second @TylerAPfledderer... there is some logic in our `Link` component that helps with the paths here... Testing this locally, the above suggestion of just using `/whitepaper-pdf/Ethereum_Whitepaper_-_Buterin_2014.pdf` seems to do the trick. This approach should work even if `/en/` is in the path still.

Reply:
@wackerow adding the language to the url is also breaking for me.

Reply:
@TylerAPfledderer @wackerow I'll submit a patch for this